packages/wrangler/src/api/pages/deploy.tsx

```diff
@@ -248,6 +267,23 @@ export async function deploy({
 		);
 	}
 
+	if (
```

Comment:
@penalosa am I correct here in thinking we only add `wrangler_config_hash` and `pages_build_output_dir` if we found a `wrangler.toml` file, valid for Pages? Shouldn't we sent an empty string otherwise? Would that be needed in order for ppl opt out of using `wrangler.toml`?

Reply:
We could send an empty string, I suppose, but omitting the properties should have the same effect

Reply:
it doesn't seem to. In fact, even if I send an empty string, it still doesn't seem to opt the project out of using the config file. Let's talk abt this IRL

Reply:
In what sense? The UI won't change, since that's not landed yet. However, the `wrangler_config_hash` property on the latest deployment config should be empty

Reply:
as in, the env vars declared in my wrangler.toml are still being picked up, even though I specifically send an empty `wrangler_config_hash`

Reply:
discussed offline, and looks like this is expected behaviour